src/components/SharedComponents/Button/LoadingButton/LoadingButton.tsx

```diff
@@ -94,7 +95,7 @@ const ButtonContent: FC<LoadingButtonProps> = (props) => {
   return (
     <>
       <OakFlex $gap="all-spacing-2" $justifyContent="center">
-        <ButtonLabel $color="white">
+        <ButtonLabel aria-live={props.ariaLive} $color="white">
```

Comment:
do we need a default? or does `undefined` equate to `off`?

Reply:
Good spot, prop best to set to off